src/solenoid/components.clj

```diff
@@ -199,3 +257,21 @@
     [(keyword (str "div#" (name id) "-value"))
      {:style {:display "none"}}
      value]]])
+
+;; PROBLEM
```

Comment:
Added a note here to think about redesigning my approach slightly. As it stands, there are many little things I have to remember to do when creating a new control type. Perhaps that's fine, because I don't expect to be adding controls forever, but it might be nice to build a better approach, so that other people could extend controls more easily/intuitively.